_content/essays/right-here-in-the-heart_boowa.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Right Here in the Heart"
+authors:
+  - boowa
+translator: geoff
+external_url: "https://www.dhammatalks.net/Books2/Maha_Boowa_Right_Here_in_the_Heart.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1B-Xjz1qEis03V1QksHD3lyJuZV6q4fhe/view?usp=sharing"
+course: thai
+tags:
+  - path
+  - dhamma
+year: 1976
+pages: 11
+
```

Comment:
Once again missing the `---` to close off the header. This is what's breaking the build here